tests/codebuild/common.sh

```diff
@@ -46,9 +55,10 @@ function wait_for_crd_status()
 # Cleans up all resources created during tests.
```

Comment:
Can we please add the required namespace parameter in the documentation here?

Reply:
Done